### PR TITLE
Add vansitha's card information

### PR DESCRIPTION
**Pull request by Vansitha**
Added my card information. 😄

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -114,6 +114,31 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```
Comment (TenTraicion) [inline index.html:117]: Kindly change `TEMPLATE: MAKE A COPY` with your username

File: index.html [hunk 2]
```diff
@@ -114,6 +114,31 @@ <h2>A project for first-time contributions</h2>
...
+              <li>
+                <a href="https://www.w3schools.com/" target="_blank" title="Has good guides and examples on how to get things done for the technology your're using.">W3Schools</a>
+              </li>
+              <li>
+                <a href="https://stackoverflow.com/" target="_blank" title="Place where all programers goto when they encouter nasty bugs.">StackOverflow</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```
Comment (TenTraicion) [inline index.html:140]: kindly change `END TEMPLATE` with your username

**Review by TenTraicion — CHANGES_REQUESTED** [review]
Kindly make those changes and commit. We will merge it!

File: index.html (hunk 2 above)
Comment (Vansitha) [reply index.html:140]: Alright. Sorry about that!

**Comment by Vansitha** [comment]
I've made the requested changes.

**Comment by TenTraicion** [comment]
@Vansitha, Thanks for your Contribution. and Congrats on your first PR Merge in this project. Your Card will be live within ~2-3 minutes, which can be found: [Here](https://syknapse.github.io/Contribute-To-This-Project/)!

**Pull request merged**